bin/runtime-common/src/refund_relayer_extension.rs

```diff
@@ -587,7 +619,7 @@ mod tests {
 	fn dispatch_info() -> DispatchInfo {
 		DispatchInfo {
 			weight: Weight::from_ref_time(
```

Comment:
is this now double so that we get `extra_weight` in tests?

Reply:
Yes, but actually it isn't required iiuc, because I'm doing the same change in that test that needs it

Reply:
reverted